xdsl/tools/command_line_tool.py

```diff
@@ -722,9 +725,9 @@ def parse_chunk(self, chunk: IO[str], file_extension: str) -> ModuleOp | None:
         try:
             return self.available_frontends[file_extension](chunk)
         except ParseError as e:
+            s = e.span
+            e.span = Span(s.start, s.end, s.input, start_offset)
             if "parsing_diagnostics" in self.args and self.args.parsing_diagnostics:
                 print(e.with_context())
             else:
                 raise Exception("Failed to parse:\n" + e.with_context()) from e
-        finally:
-            chunk.close()
```

Comment:
Do we no longer want this? Why would this not be needed?

Reply:
Good catch, forgot this change, it can stay!
Note that we never needed it, it's just cleaner file/memory management